app/code/community/Aligent/CacheObserver/Model/Observer.php

```diff
@@ -38,9 +35,11 @@ public function customBlockCache(Varien_Event_Observer $observer) {
                     return $this;
                 }
             }
+            $iCacheLifetime = Mage::getStoreConfig(self::CACHE_LIFETIME_CONFIG);
```

Comment:
@moloughlin This observer method gets called many times in the course of rendering a page, and most of the blocks we observe aren't made cacheable by cacheobserver.  However we're doing this lookup into the XML tree for the cache lifetime on every block regardless of whether we subsequently act on the block.  The observer class is instantiated as a singleton.  Can we make $iCacheLifetime a class level member variable and set it in the constructor instead?  That way we only make one XML lookup for the entire page load.